.travis.yml

```diff
@@ -12,6 +12,5 @@ cache:
     - node_modules
 
 script:
-  - yarn lint
-  - yarn test
+   - yarn run jest --projects jest.*.config.js
```

Comment:
This is a follow up on https://github.com/commercetools/ui-kit/pull/13#issuecomment-418364019